app/templates/profile.hbs

```diff
@@ -233,7 +233,7 @@
                             city_stat=d.poverty_rate_nyc}}
             of residents have incomes below the <a href="http://www1.nyc.gov/site/opportunity/poverty-in-nyc/poverty-measure.page" target="_blank">NYCgov poverty threshold</a>. Compare to <a href="https://www1.nyc.gov/assets/planning/download/office/data-maps/nyc-population/acs/econ_2016acs5yr_puma.xlsx?r=1" target="_blank">federal poverty&nbsp;rate</a>.
           {{/data.indicator}}
-        {{/key-indicators}}
+        {{/key-indicators}}{{!change this link? ^^}}
```

Comment:
Is this a comment?

Reply:
Yeah -- getting an answer from Chris right now on whether we should update this too. You can ignore for now, it'll be gone in the merge